python/cudf/cudf/core/column/struct.py

```diff
@@ -150,6 +150,8 @@ class StructMethods(ColumnMethods):
     Struct methods for Series
     """
 
+    _column: StructColumn
```

Comment:
Did we just not run into this before because no methods were accessing the attribute? All subclasses of `ColumnMethods` should have an `_column` attribute of the corresponding type, right?

Reply:
There's no AttributeError if we don't include this; this is really there more to help typing (`mypy`) along.